### PR TITLE
Style

### PR DESCRIPTION
**Pull request by michelle-gu**

### CODE REVIEW

File: src/css/twitter.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+a {
+	color: black;
+	text-decoration: none;
+}
+
+.post {
+	font-size: 16px;
+	border-width: 1px;
```
Comment (kev-chan) [inline src/css/twitter.css]: shorthand this

File: src/css/twitter.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	align-items: center;
+}
+
+.user {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	flex: 0 0 150px;
+	padding-left: 22px;
```
Comment (kev-chan) [inline src/css/twitter.css]: shorthand

File: src/css/twitter.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+#timeline-button-div {
+	padding-left: 50px;
+}
+
+#timeline-button {
+	font-size: 12px;
+	border-radius: 25px;
+	padding-left: 12px;
```
Comment (kev-chan) [inline src/css/twitter.css]: shorthand

File: src/css/twitter.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	padding-bottom: 4px;
+}
+
+#timeline {
+	border-style: solid;
+	border-width: 1px;
+}
+
+#timeline-div {
+	padding-left: 50px;
```
Comment (kev-chan) [inline src/css/twitter.css]: shorthand

File: src/js/twitter.js [hunk 5]
```diff
@@ -23,36 +23,66 @@ function buildTimeline(responseText) {
...
+
+		// Create post
+		var postDiv = document.createElement("div");
+		postDiv.className = "post"
+		postDiv.appendChild(userDiv);
+		postDiv.appendChild(tweetLink);
+		// Alternate colors
 		if (i % 2 == 0) {
-			postDiv.style.backgroundColor = "red";
+			postDiv.style.backgroundColor = "#e8f5fd";
```
Comment (kev-chan) [inline src/js/twitter.js]: use css for this

**Comment by michelle-gu** [comment]
from Kevin:
- there is an additional gray border around each post, only thing needed is the separator between posts which is currently 2px. Let’s make it 1px
- shrink the image by 5 px all around
- reduce overall font-size by a pixel
- reduce Get Timeline button padding by 2 px horizontally each side, increase 1 vertically on each side

**Pull request merged**